analyses/tp53_nf1_score/05-tp53-altered-annotation.Rmd

```diff
@@ -433,6 +460,23 @@ ggplot(tp53_alterations_score_wide, aes(x = factor(tp53_altered), y = tp53_score
   xlab("tp53 altered status") +
   facet_wrap(.~cancer_predispositions)
 
+# plot lfs for manuscript
```

Comment:
I think that this is because there are only two samples there and I am not sure how many are needed for the violin - I was playing around with adding jitter as well - maybe I should go back to this one since there are fewer samples. In the plot with _all_ samples, the `Other` jitter is too overwhelming. Let me play with that.

Reply:
If there are only two samples, we should definitely show the individual points.

Reply:
Ahh, that reminds me, we should either print or record all of the group Ns somewhere to be able to describe in legends.

Reply:
Added in [504d078](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1152/commits/504d078673ea299d41b7765c9d74ef68929e24c5)